### PR TITLE
Aws sdk v2

### PR DESCRIPTION
**Pull request by mguymon**
@dstuebe @peakxu 

The future is now. `aws sdk` V2 supports tagging, so we can drop our dep on fog finally.

### CODE REVIEW

File: lib/pipely/deploy/json_definition.rb [hunk 1]
```diff
@@ -9,21 +10,22 @@ module Deploy
     #
     class JSONDefinition
       def self.parse(definition)
-        definition_objects = JSON.parse(definition)['objects']
+        definition_objects =
+          JSON.parse(definition)['objects'].map(&:deep_symbolize_keys)
```
Comment (dstuebe) [inline lib/pipely/deploy/json_definition.rb]: json parse supports a symbolize keys option

**Comment by dstuebe** [comment]
What pipelines have you tested compatibility on?

**Comment by mguymon** [comment]
so long fog and thanks for all the fish

**Comment by dstuebe** [comment]
Woot!

**Comment by peakxu** [comment]
+1 on successful pipeline deploy + graph w/ this.

**Pull request merged**